hubblestack/extmods/grains/docker_details.py

```diff
@@ -0,0 +1,76 @@
+"""
+HubbleStack Docker Details Grain.
+CLI Usage - hubble grains.get docker_details
+Example Output - {u'installed': True, u'running': True}
+Author - Mudit Agarwal (muagarwa@adobe.com)
+"""
+import salt.utils.platform
+import logging
+from hubblestack.utils.osquery_lib import query as osquery_util
+log = logging.getLogger(__name__)
+
+def get_docker_details(grains):
+  try:
```

Comment:
I actually posted these comments a week ago, I thought. I failed to hit submit again.

I don't think this is a huge problem or anything, but it might be worth looking at regardless.

Reply:
removed try/except.